Numbers.java

```diff
@@ -0,0 +1,82 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Numbers {
+    Scanner object = new Scanner(System.in);

```

Comment:
You should avoid naming object references by any potential java keywords (in this case object).
object could be input for example